cassandra-protocol/src/types/cassandra_type.rs

```diff
@@ -0,0 +1,34 @@
+use num::BigInt;
+use std::collections::HashMap;
+use std::net::IpAddr;
+
+use super::prelude::{Blob, Decimal};
+
+#[derive(Debug)]
```

Comment:
Could use more derives like `PartialEq` etc.

Reply:
Added `PartialEq` and `Clone`